src/components/Eval.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+export default class Eval extends React.Component {
+  onSubmit = () => {
+    let num1F = Number(this.props.num1);
+    let num2F = Number(this.props.num2);
```

Comment:
Why 'num1F' and 'num2F' instead of 'num1' and 'num2'?

Reply:
Good point on the divide by zero.

I thought it may be bad practice to have two variables having the same name, even if one is in props and one is in the method. If it's not bad practice I'll start doing that